test/models/host_test.rb

```diff
@@ -266,6 +266,26 @@ def teardown
       host.expects(:enable_orchestration!).never
       assert host.import_facts(raw['facts'])
     end
+
+    test 'should import facts boot time to report data facet' do
+      refute Host.find_by_name('sinn1636.lan')
+      raw = read_json_fixture('facts/facts_with_certname.json')
+      first_boot_time = nil
+      freeze_time do
+        host = Host.import_host(raw['name'], 'puppet')
```

Comment:
you should define the host variable outside of the block. It is undefined in the other one.

Reply:
yeah I did the same for first_boot_time already, importing needs to happen while time is frozen... feel free to wait with the review until tests are green, I'm monitoring jenkins output here :-)